src/scripts/modules/media/body/body.coffee

```diff
@@ -469,7 +469,12 @@ define (require) ->
     toggleSolution: (e) ->
       $solution = $(e.currentTarget).closest('.solution, [data-type="solution"]')
       $solution.toggleClass('ui-solution-visible')
-
+      if $solution.hasClass('ui-solution-visible')
```

Comment:
The first change is ok. But the problem is that if I follow the second change, the button will only be able to be cliked once. After that, however I clicked the button, the solution will neither display nor hide. I don't know why such a weired thing will happen.